### ISSUE TITLE
Dark Theme 

### ISSUE THREAD

**Issue by PatrickWinkler** [DrKLO/Telegram#982]
An Dark Theme option would be great.
This would have to purposes, less battery consumption and its more comfortable to the eyes when you write at night.

**Comment by Multimoon** [comment]
I second this would be nice.

**Comment by Sqwrly** [comment]
Definitely needs some themes.

**Comment by woj-tek** [comment]
second that. some apps have 'material dark', and 'material black', which should work.

**Comment by joshery420** [comment]
This is one of my only real complaints with this app.
A nice dark grey Material theme would be the best.

**Comment by adunaphel13** [comment]
Yeah, material dark for Android. The white background hurts my eyes.

No need for full theming, I think. Just a dark option.

**Comment by Multimoon** [comment]
@DrKLO has said he won't do any themeing, but hopefully he'll at least do a dark theme.

**Comment by akaleeroy** [comment]
Dark theme, yes. :+1:
Please consider that on OLED screens a dark theme can save a not insubstantial amount of battery! 

> You can see pretty clearly that in this case, a black screen does indeed reduce how much juice the phone is using. Power consumption was **41% lower** overall when using the predominantly black interface in Reddit Sync.
>  
> So what does this mean for your phone's battery life? You can definitely make a dent in battery drain with predominantly black apps. If you use a device heavily, the display will be at the top of the battery usage list, so the impact can be noticeable if you keep to AMOLED-friendly apps most of the time.  
[*How much power does a black interface really save on AMOLED displays? | Greenbot*](http://www.greenbot.com/article/2834583/how-much-power-does-a-black-interface-really-save-on-amoled-displays.html)

If you're going to be spending a lot of time in the app (and by God you have all the reasons, nice job!) this is more than a nice-to-have. Also, just inverting colors on Android would mess with the pictures.
